docs/src/CommutativeAlgebra/localizations.md

```diff
@@ -15,45 +15,55 @@ Pages = ["localizations.md"]
-rings, as well as localizations of affine algebras, at various types of multiplicatively closed subsets. For the convenience of
-the developer, in an appendix, we outline a general framework for creating new concrete instances of localized
-rings in OSCAR, describing relevant abstract types and indicating the functionality to be implemented.
-
-!!! note
-    All OSCAR types discussed in this section are parametrized. For simplicity of the presentation, we omit corresponding details.
-
-Most functions for handling localizations of multivariate polynomial rings rely on Gröbner bases. More precisely,
-if ``R`` is a multivariate polynomial ring, Gröbner bases allow one to solve  ideal- and module-theoretic
-questions concerning ``R[U^{-1}]`` by computations over ``R``.
-
-!!! note
-    Recall that in OSCAR, Gröbner bases are implemented for multivariate polynomial rings over fields (exact fields supported by OSCAR) and for multivariate polynomial rings over the integers.
+Write ``\frac{r}{u}`` for the equivalence class of ``(r, u)`` and ``R[U^{-1}]`` for the set of all equivalence classes.
+Mimicking the standard arithmetic for fractions, ``R[U^{-1}]`` can be made into a ring. This ring is called the
+*localization of ``R`` at ``U``*. It comes equipped with  the natural ring homomorphism
+```math
+\iota : R\rightarrow R[U^{-1}],\; r \mapsto \frac{r}{1}.
+```
+Given an ``R``-module ``M``, the analogous construction yields an ``R[U^{-1}]``-module ``M[U^{-1}]`` which is
+called the *localization  of ``M`` at ``U``*.  See, for instance, [Eis95](@cite).
+
+OSCAR supports several types of multiplicatively closed subsets of multivariate polynomial rings.
+The related functionality is not only used to construct localizations of
+- multivariate polynomial rings (OSCAR type `MPolyRing`),
+but also to construct localizations of
+- quotients of multivariate polynomial rings (OSCAR type `MPolyQuo`).
+
+For the latter, recall that if ``A`` is a quotient of a multivariate polynomial ring ``R``,
+``p: R \rightarrow A`` is the projection map, and ``S`` is a multiplicatively closed subset of ``A``,
+then the preimage $U = p^{-1}(S)$ is a multiplicatively closed subset of ``R`` such that ``S = p(U)``.
+Creating the localization of the ring ``A`` at ``S `` in OSCAR amounts to localize the ``R``-module
+``A`` at ``U``, and equip the result with the appropriate ring structure. Hence, the corresponding
+constructor takes as input ``A`` and ``U`` rather than ``A`` and ``S``.
 
 !!! note
-    In the local context, following Hironaka and Grauert, it is often common to use the name *standard basis* instead of Gröbner basis. See, for example, [GP08](@cite).
-	
+    Most functions described in this section rely on the computation of standard bases. Recall that OSCAR supports standard
+    bases for multivariate polynomial rings over fields (exact fields supported by OSCAR) and for multivariate
+	polynomial rings over the integers.	
```

Comment:
But for general localizations, the standard bases play only a minor role in a very particular subcase. I would probably not put this so prominently here.